webknossos/webknossos/dataset/_utils/pims_images.py

```diff
@@ -496,6 +498,7 @@ def copy_to_view(
             with mag_view.get_buffered_slice_writer(
                 relative_offset=(0, 0, z_start * mag_view.mag.z),
                 buffer_size=mag_view.info.chunk_shape.z,
+                json_update_allowed=False,
```

Comment:
please add a comment that `copy_to_view` is typically used in a multiprocessing-context and therefore needs this flag.

Reply:
Is done with commit ea7dc34.